### PR TITLE
Tng 764

### PR DESCRIPTION
**Pull request by rrieger-espirit**
in this ticket i added tests for sections. NewsTeaserSections is not finished yet, this will be fixed by another time

### CODE REVIEW

**Review by tgolbs — CHANGES_REQUESTED** [review]

File: src/components/Sections/HeaderSection/Breadcrumbs/__tests__/Breadcrumbs.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { render, fireEvent } from "@testing-library/vue";
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/Sections/HeaderSection/Breadcrumbs/__tests__/Breadcrumbs.spec.tsx:1]: Hat Robin mit "checks if all items have been rendered and are in the right order" getan.

File: src/components/Sections/HeaderSection/Breadcrumbs/__tests__/Breadcrumbs.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        ],
+      },
+    });
+
+    const breadcrumb = container.querySelector(
+      ".Breadcrumbs--Item [data-testid='item-1']",
+    );
+    console.log(breadcrumb);
+    await fireEvent(breadcrumb!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/Sections/HeaderSection/Breadcrumbs/__tests__/Breadcrumbs.spec.tsx:38]: Das wird jetzt gemacht mit expect(spy.mock.calls[0][0]).toEqual

File: src/components/Sections/HeaderSection/__tests__/HeaderSection.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { render, fireEvent } from "@testing-library/vue";
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/Sections/HeaderSection/__tests__/HeaderSection.spec.tsx:1]: add test it("correctly consumes the specified background image"...)

File: src/components/Sections/TeaserSection/__tests__/TeaserSection.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render } from "@testing-library/vue";
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/Sections/TeaserSection/__tests__/TeaserSection.spec.tsx:1]: done

File: src/components/Sections/TeaserSection/__tests__/TeaserSection.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    const buttonText = "button";
+    const { getByTestId } = render(TeaserSection, {
+      props: {
+        headline: headlineTest,
+        kicker: kickerTest,
+        text: textTest,
+        buttonText: buttonText,
+      },
+    });
+    const headline = getByTestId("teasersection-headline");
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/Sections/TeaserSection/__tests__/TeaserSection.spec.tsx:18]: done

**Review by tgolbs — APPROVED** [review]

**Review by vlapl — APPROVED** [review]
Tobi did the review this morning (with me watching). Now I fixed a typo, as agreed upon, and will proceed merging this pr.

**Pull request merged**